nannou/src/window.rs

```diff
@@ -241,6 +243,8 @@ pub struct Window {
     pub(crate) frame_count: u64,
     pub(crate) user_functions: UserFunctions,
     pub(crate) tracked_state: TrackedState,
+    pub(crate) is_invalidated: bool,
```

Comment:
I named it this because "invalidate" is often used in GUIs to indicate that a window must be redrawn (e.g. [Windows](https://docs.microsoft.com/en-us/windows/win32/gdi/invalidating-the-client-area)). I'll add a comment for now, but if you prefer the other name let me know and I'll switch it!